### PR TITLE
Create neocities-learn.md

### PR DESCRIPTION
**Pull request by jsaji06**

### CODE REVIEW

File: articles/neocities-learn.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+author: ele
```
Comment (MineRobber9000) [inline articles/neocities-learn.md]: Use your full site name. Neowiki [links its contributors](https://minerobber.neocities.org/neowiki/writing-for-neowiki.html) and it cannot do this if you use a nickname.

File: articles/neocities-learn.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - Twitter Bootstrap
+  - Pure
+  - Bass Tracks
+  - Min
+  
+## Tutorials
+Neocities has their own crash course featuring Penelope the Cat. There are 10 parts, covering from putting text on a page, to making links.
+
+Regarding other things, there are resources that allow you to learn them. Neocities currently does not have a tutorial for these.
+
```
Comment (MineRobber9000) [inline articles/neocities-learn.md]: Link the tutorial and I'll merge.

**Review by MineRobber9000 — CHANGES_REQUESTED** [review]
All in all, not too bad. Just fix these few things and you'll be golden!

File: articles/neocities-learn.md (hunk 1 above)
Comment (jsaji06) [reply articles/neocities-learn.md]: alright

File: articles/neocities-learn.md (hunk 2 above)
Comment (jsaji06) [reply articles/neocities-learn.md]: Okay.

**Review by MineRobber9000 — APPROVED** [review]

**Comment by MineRobber9000** [comment]
Thanks!

**Pull request merged**